src/api/cpp/cvc5.h

```diff
@@ -2840,8 +2840,12 @@ class CVC5_EXPORT Stat
   friend std::ostream& operator<<(std::ostream& os, const Stat& sv);
   /** Representation of a histogram: maps names to frequencies. */
   using HistogramData = std::map<std::string, uint64_t>;
-  /** Can only be obtained from a `Statistics` object. */
-  Stat() = delete;
+  /**
+   * Create an empty statistics object. None of the ``isX()``and ``getX()``
```

Comment:
Perhaps add assertions in such functions?

Reply:
Yeah, I guess we should do that.